actors/builtin/verifreg/verified_registry_state.go

```diff
@@ -37,8 +52,71 @@ func ConstructState(store adt.Store, rootKeyAddress addr.Address) (*State, error
 	}
 
 	return &State{
-		RootKey:         rootKeyAddress,
-		Verifiers:       emptyMapCid,
-		VerifiedClients: emptyMapCid,
+		RootKey:                  rootKeyAddress,
+		Verifiers:                emptyMapCid,
+		VerifiedClients:          emptyMapCid,
+		RemoveDataCapProposalIDs: emptyMapCid,
 	}, nil
 }
+
+// A verifier who wants to send/agree to a RemoveDataCapRequest should sign a RemoveDataCapProposal and send the signed proposal to the root key holder.
```

Comment:
I don't feel like i understand entirely how this (datacap permissioning threat model I guess??) works, so more eyes on this section of the logic might be good if there's uncertainty?

Reply:
Yah this model is confirmed by the fil+ team.